index.js

```diff
@@ -239,6 +240,17 @@ Swarm.prototype.destroy = function() {
 	});
 };
 
+Swarm.prototype.reconnectAll = function() {
+	var self = this;
+	Object.keys(this._peers).forEach(function(addr) {
+		var isConnected = self.wires.some(function (e) { if (e.peerAddress == addr) return true; });
```

Comment:
`if (e.peerAddress == addr) return true;`
is equivalent to
`return e.peerAddress === addr;`